doc/py_tutorials/py_feature2d/py_fast/py_fast.markdown

```diff
@@ -82,7 +82,7 @@ Non-maximum Suppression.
 
 It is several times faster than other existing corner detectors.
 
-But it is not robust to high levels of noise. It is dependant on a threshold.
+But it is not robust to high levels of noise. It is dependent on a threshold.
```

Comment:
Right? May be "It depends ..."? (English is not my native language)

Reply:
@alalek choose one:   
`It is dependent on a threshold`  
`It depends on a threshold`

Reply:
OK, lets leave as is.